build.sbt

```diff
@@ -73,6 +73,13 @@ lazy val `client-okhttp` = project
   .settings(clientOkHttpSettings)
   .disablePlugins(ScriptedPlugin)
 
+lazy val `client-cache` = project
+  .in(file("modules/client-cache"))
+  .settings(moduleName := "frees-rpc-client-cache")
+  .settings(clientCacheSettings)
+  .settings(addCompilerPlugin(%%("paradise") cross CrossVersion.full))
```

Comment:
Without this line I get a `Missing required plugin: macroparadise` compiler error.

Reply:
It's weird, it should be provided by the project plugin: https://github.com/frees-io/freestyle-rpc/blob/14671779136aa05ea146ae3126ef410cbd4f52db/project/ProjectPlugin.scala#L209

Reply:
I was overriding the `libraryDependencies` before, but it looks like we only get the paradise plugin and a scalatest test dependency.
I switched back to `++=`, so should be fine now.